### PR TITLE
Replace Element links with Matrix page

### PR DESCRIPTION
**Pull request by HardenedSteel**
Not everyone using Element for Matrix, with matrix.to links people can choose a client.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ If you’d like to add a cryptocurrency to BasicSwap, either [apply for a listin
 
 ### Chats
 
-* **For developers** The chat [#particl-dev:matrix.org](https://app.element.io/#/room/#particl-dev:matrix.org) using [Element](https://element.io) (formerly Riot).
+* **For developers** The chat [#particl-dev:matrix.org](https://matrix.to/#/#particl-dev:matrix.org) using a Matrix client.
 * **For community** The community chat [https://discord.me/particl](https://discord.me/particl) [![Discord](https://img.shields.io/discord/391967609660112925)](https://discord.me/particl).
```
Comment (nahuhh) [inline README.md:104]: Should link to basicswap rooms

File: README.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ For non-developers curious to explore a new world of commerce, binaries can be d
 
 * [Telegram](https://t.me/particlhelp)
 
-* [Element](https://app.element.io/#/room/#particlhelp:matrix.org)
+* [Matrix](https://matrix.to/#/#particlhelp:matrix.org)
```
Comment (nahuhh) [inline README.md:121]: Same

**Review by nahuhh — COMMENTED** [review]
s/particl/basicswap

File: README.md (hunk 1 above)
Comment (HardenedSteel) [reply README.md:104]: I don't know their rooms. I can fix if someone knows.

Comment (nahuhh) [reply README.md:104]: This one can probably stay the same(dev)

Comment (nahuhh) [inline README.md:104]: add these i think (to community)

general
https://matrix.to/#/%23particl:matrix.org
basicswap
https://matrix.to/#/%23basicswap:matrix.org

**Pull request merged**